engine/flink/components/dev-table/src/main/scala/pl/touk/nussknacker/engine/flink/table/utils/RowToTestDataConverter.scala

```diff
@@ -0,0 +1,20 @@
+package pl.touk.nussknacker.engine.flink.table.utils
+
+import org.apache.flink.types.Row
+import pl.touk.nussknacker.engine.api.test.{TestData, TestRecord}
+import pl.touk.nussknacker.engine.flink.table.TableDefinition
+import pl.touk.nussknacker.engine.util.json.BestEffortJsonEncoder
+
+import scala.collection.immutable.ListMap
+
+class RowToTestDataConverter(classLoaderForEncoder: ClassLoader) {
+
+  private val encoder = new BestEffortJsonEncoder(failOnUnknown = false, classLoader = classLoaderForEncoder)
+
+  def rowsToTestData(rows: List[Row], tableDefinition: TableDefinition): TestData = TestData(rows.map(row => {
```

Comment:
Taking the `tableDefinition` allows for deterministic and controllable ordering of fields. This way its always in the same order as columns in the CREATE TABLE statement.